docs/CHANGELOG.md

```diff
@@ -66,7 +70,7 @@ title: Changelog
 
 * Add test case for conflict with internal `@variant` variable.
 
-   *David Backeus*
+    *David Backeus*
```

Comment:
looks like this change is irrelevant to the PR?

Reply:
Yes, I was just fixing the changelog indentation at the same time

Reply:
Ok, normally in that case we should add these "unrelated" changes to a separate commit, like "Fix indentation"

Reply:
We squash commits on merge, so I'm not too worried about atomic commits unless it's a big PR that might be best reviewed step-wise. Even then, we'd probably want to multiple PRs ❤️